packages/types/src/asset-types.ts

```diff
@@ -179,3 +179,12 @@ export interface Asset {
   /** Denom key of variant of asset this is grouped with. */
   variantGroupKey?: string;
 }
+
+export type MinimalAsset = KeplrBaseCurrency & {
```

Comment:
If I were to implement this, given our divorce from Keplr, I'd just inline the members here as we had and avoid importing their types